client/src/game/models/shapes.ts

```diff
@@ -96,8 +96,8 @@ export interface ServerTracker {
     value: number;
     maxvalue: number;
     draw: boolean;
-    primary_color: string;
-    secondary_color: string;
+    primaryColor: string;
+    secondaryColor: string;
```

Comment:
This should actually remain snake_cased :D because this is the representation straight from the server, it's the only exception where the client code uses snake_case.

There is conversion code for auras to convert to and from server representation.

Reply:
whoops. No problem, I'll look up the conversion code from the auras and do something similar with the trackers

Reply:
also better to ask this now, would you prefer to use 'colour' or 'color'? I went with color as it's most likely to be consistent with other libraries and code but I see you've used colour in the auras so can change it to be consistent with that if you prefer.

Reply:
I don't mind super hard, but I personally enjoy the British variant more, but the American is more common so that's fine as well.